demo/src/screens/incubatorScreens/SvgImageScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import {Image, View, Text} from 'react-native-ui-lib';
+// @ts-ignore
+import {renderBooleanOption, renderRadioGroup} from '../ExampleScreenPresenter';
+const file = require('../../../../uilib-docs/src/images/newDesign/headerLogo.svg').default;
```

Comment:
Consider loading this asset as any other image asset here
`./demo/src/configurations.js`

Reply:
Done